mkdocs/__main__.py

```diff
@@ -18,6 +18,11 @@
 
 log = logging.getLogger(__name__)
 
+# Disable the warning that Click displays (as of Click version 5.0) when users
+# use unicode_literals in Python 2.
+# See http://click.pocoo.org/dev/python3/#unicode-literals for more details.
+click.disable_unicode_literals_warning = True
+
```

Comment:
You appear to not be working with the latest in master. These lines were removed in a recent commit (e092584c). They should not be here. I would recommend rebasing against the latest in master.